mmpy_bot/event_handler.py

```diff
@@ -48,8 +48,7 @@ async def _check_queue_loop(self, webhook_queue: queue.Queue):
                 event = webhook_queue.get_nowait()
                 await self._handle_webhook(event)
             except queue.Empty:
-                pass
-            await asyncio.sleep(0.0001)
+                await asyncio.sleep(0.5)
```

Comment:
Did you really intend to move the sleep only for the exception?

Reply:
Yes. When the queue is not empty, handling messages without blocking maybe not a bad idea.